apps/src/aichat/types.ts

```diff
@@ -13,6 +13,8 @@ export type AichatInteractionStatusValue = string;
 export interface ChatEvent {
   // UTC timestamp in milliseconds
   timestamp: number;
+  // This field is optional but when it is defined, it must be set to `true`.
```

Comment:
Added comment - addresses https://github.com/code-dot-org/code-dot-org/pull/60053#discussion_r1698884071 @sanchitmalhotra126

Reply:
Can you include the why? (to avoid bloating all ChatEvents since only a small subset will have this field set to true)

Reply:
I think I do in the line below.

Reply:
OH sorry! I didn't see it under the comment dialog 🤦🏻‍♀️

Reply:
Lmk if you'd like me to add more details. Thanks!

Reply:
No that is perfect